api/hasura/cron/check-nominee.ts

```diff
@@ -0,0 +1,75 @@
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+
+import { gql } from '../../../api-lib/Gql';
+import { sendSocialMessage } from '../../../api-lib/sendSocialMessage';
+
+export default async function handler(req: VercelRequest, res: VercelResponse) {
+  try {
+    const { nominees } = await gql.q('query')({
+      nominees: [
```

Comment:
shared libraries

Reply:
Not following, should this query be moved somewhere else or?

Reply:
yes, I gave more details above